test/snippet/alignment/cigar_conversion/cigar_from_alignment_with_clipping.cpp

```diff
@@ -0,0 +1,29 @@
+#include <seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp>
+#include <seqan3/alignment/cigar_conversion/cigar_from_alignment.hpp>
+#include <seqan3/alignment/decorator/gap_decorator.hpp>
+#include <seqan3/alphabet/gap/gapped.hpp>
+#include <seqan3/alphabet/nucleotide/dna5.hpp>
+#include <seqan3/core/debug_stream.hpp>
+#include <seqan3/utility/views/slice.hpp>
+
+int main()
+{
+    using namespace seqan3::literals;
+
+    seqan3::dna5_vector reference = "ATGGCGTAGAGCTTCCCCCCCCCCCCCCCCC"_dna5;
+    seqan3::dna5_vector read = "ATGCCCCGTTGCTT"_dna5; // length 14
+
+    // Let's say, we want to ignore the last 2 bases of the query because the quality is low
+    // we thus only align the first 12 bases: (the last to will be soft-clipped bases in the CIGAR string)
```

Comment:
```suggestion
    // Let's say, we want to ignore the last 2 bases of the query because the quality is low.
    // We thus only align the first 12 bases, the last two will be soft-clipped bases in the CIGAR string.
```